modules/auxiliary/scanner/http/cisco_ironport_enum.rb

```diff
@@ -61,6 +61,7 @@ def check_conn?
         'method'    => 'GET'
       })
       print_good("#{rhost}:#{rport} - Server is responsive...")
+      return 1
     rescue ::Rex::ConnectionRefused, ::Rex::HostUnreachable, ::Rex::ConnectionTimeout, ::Rex::ConnectionError, ::Errno::EPIPE
       return
```

Comment:
Yeah, that's just a print, though, and the print should actually be conditional on the success or failure of the ```send_request_cgi```. :)